### PR TITLE
baseURI funcs + test

### PR DESCRIPTION
**Pull request by SergeyMushta**

### CODE REVIEW

File: contracts/NftInjectableTokenIdRolesUpdatableBaseURI.sol [hunk 1]
```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
```
Comment (KasparPeterson) [inline contracts/NftInjectableTokenIdRolesUpdatableBaseURI.sol:2]: can we use the latest version instead of floating?

**Review by KasparPeterson — DISMISSED** [review]

File: contracts/NftInjectableTokenIdRolesUpdatableBaseURI.sol (hunk 1 above)
Comment (kristjanpeterson1) [reply contracts/NftInjectableTokenIdRolesUpdatableBaseURI.sol:2]: 0.8.9 as other contracts

File: contracts/NftInjectableTokenIdRolesUpdatableBaseURI.sol [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+import "@openzeppelin/contracts/access/AccessControl.sol";
+
+contract NftInjectableTokenIdRolesUpdatableBaseURI is ERC721URIStorage, AccessControl {
+    bytes32 public constant MINTER_ROLE = keccak256("MINTER_ROLE");
+    address private _owner;
+
+    bool public isFreezeTokenUris;
+    mapping (uint256 => bool) public freezeTokenUris;
+
+    string private baseURI;
```
Comment (kristjanpeterson1) [inline contracts/NftInjectableTokenIdRolesUpdatableBaseURI.sol:14]: This is used when tokenURI function from ERC721URIStorage.sol is called yeah?

Comment (SergeyMushta) [reply contracts/NftInjectableTokenIdRolesUpdatableBaseURI.sol:14]: check function _baseURI() - it overrides default function (which just returned "" by default)

Comment (kristjanpeterson1) [reply contracts/NftInjectableTokenIdRolesUpdatableBaseURI.sol:14]: ah yes,  i see from tests

File: contracts/NftInjectableTokenIdRolesUpdatableBaseURI.sol (hunk 1 above)
Comment (SergeyMushta) [reply contracts/NftInjectableTokenIdRolesUpdatableBaseURI.sol:2]: I think yes, will give a try

File: contracts/NftInjectableTokenIdRolesUpdatableBaseURI.sol (hunk 2 above)
Comment (kristjanpeterson1) [reply contracts/NftInjectableTokenIdRolesUpdatableBaseURI.sol:14]: Oh very cool nice

File: contracts/NftInjectableTokenIdRolesUpdatableBaseURI.sol (hunk 1 above)
Comment (SergeyMushta) [reply contracts/NftInjectableTokenIdRolesUpdatableBaseURI.sol:2]: https://hardhat.org/reference/solidity-support.html - latest supported Solidity is 0.8.9 for now, 0.8.10 isn't supported by hardhat yet. So will set to 0.8.9 in order to not break infrastructure

**Review by kristjanpeterson1 — APPROVED** [review]

**Pull request merged**